dev/ProgressBar/ProgressBar.xaml

```diff
@@ -135,7 +154,7 @@
                                         <ColorAnimation
                                             Storyboard.TargetName="DeterminateProgressBarIndicator"
                                             Storyboard.TargetProperty="(Shape.Fill).(SolidColorBrush.Color)"
-                                            To="{ThemeResource SystemErrorTextColor}"
```

Comment:
why is this change necessary?

Reply:
According to #1766 we should use `SystemControlErrorTextForegroundBrush` which is a SolidColorbrush. However ColorAnimation does not support brushes, but expects us to pass a Color object, that's why we need to use the binding now.

Reply:
@YuliKl can you comment? This seems like it's going to create 2 extra allocations (at a minimum - one for the `Binding` and one for the `Brush`) when I don't think it's necessary. Seems like having a `ThemeResource` to `SystemErrorTextColor` would do the trick just fine?

Reply:
Sorry I missed this question earlier. A potential problem I foresee with using a Color resource is what happens with the control is running in High Contrast. SystemControlErrorTextForegroundBrush falls back to SystemColorWindowTextColor in HC (which can be modified by the user), but SystemErrorTextColor retains its static value of #FFF000. If we hardcode a color, I think we might get the appearance wrong in HC.

Reply:
looking at generic.xaml the high contrast version of the brush should be 
            <SolidColorBrush x:Key="SystemControlErrorTextForegroundBrush" Color="{ThemeResource SystemColorWindowTextColor}" />

---
In reply to: [489609371](https://github.com/microsoft/microsoft-ui-xaml/pull/3201#discussion_r489609371) [](ancestors = 489609371)

Reply:
So yes, there is a tangible difference between the brush and the color, and I think its worth keeping the binding.

---
In reply to: [491100783](https://github.com/microsoft/microsoft-ui-xaml/pull/3201#discussion_r491100783) [](ancestors = 491100783,489609371)